### ISSUE TITLE
How to connect to vps

### ISSUE THREAD

**Issue by SutSug** [G047/Agar.io-Bots#52]
How do I connect the files to a vps

**Comment by MasonBurdette** [comment]
Use a FTP application such as FileZilla to install the files. Then use Putty or another SSH client to run the actual project.

**Comment by SutSug** [comment]
@MasonBurdette / dosk can out give skype for a contact I need to ask a few questions privately

**Comment by MasonBurdette** [comment]
@SutSug sorry I don't want to be bombarded with messages as my skype is private.. You could give me yours tho and I'd possibly get in touch.

**Comment by SutSug** [comment]
@masonburdette I added u was that u my display name is SutSug

**Comment by fayizan** [comment]
@MasonBurdette is this project works still

**Comment by k69sknk** [comment]
Yup need to know that is it working ? will share simple installation tutorial for helping who wants.

**Comment by SutSug** [comment]
@k69sknk do ur bots connect

**Comment by k69sknk** [comment]
Only 1 bot... something wrong with settings.

**Comment by SutSug** [comment]
@k69sknk do u switch proxies to http proxies because that might be the problem since only 1 connects

**Comment by k69sknk** [comment]
How to switch to http ? just get http proxies ? all my proxies are http... thx
